Entities/Industry/Drill/Drill.as

```diff
@@ -254,34 +262,48 @@ void onTick(CBlob@ this)
 							f32 attack_dam = 1.0f;
 							HitInfo@ hi = hitInfos[i];
 							bool hit_constructed = false;
-							if (hi.blob !is null) // blob
+							CBlob@ b = hi.blob;
+							if (b !is null) // blob
 							{
+								// blob ignore list, this stops the drill from overheating f a s t
+								// or blobs to increase damage to
+								int nhash = b.getName().getHash();
+								switch(nhash)
+								{
+									case 575725963:   // mat_stone
+									case -282477713:  // mat_wood
+									case -1370030172: // mat_gold
+									{
+										continue;     // we dont want to damage them
```

Comment:
does this even work as intended in AS? in C++ this would do nothing because the `continue;` would apply to the switch and do nothing